README.md

```diff
@@ -29,7 +29,7 @@ TEAMMATES team wishes to thank the following invaluable contributions:
 * [**School of Computing, National University of Singapore (NUS)**](http://www.comp.nus.edu.sg), for providing us with the infrastructure support to run the project.
 * [**Centre for Development of Teaching and Learning (CDTL)**](http://www.cdtl.nus.edu.sg/) of NUS, for supporting us with several *Teaching Enhancement Grants* over the years.
 * **Learning Innovation Fund-Technology (LIF-T)** initiative of NUS, for funding us for the 2015-2018 period.
-* **Google Summer of Code** Program, for including TEAMMATES as a mentor organization in *GSoC2014*, *GSoC2015*, *GSoC2016*, and *GSoC2017* editions.
+* **Google Summer of Code** Program, for including TEAMMATES as a mentor organization in *GSoC2014*, *GSoC2015*, *GSoC2016*, *GSoC2017*, and *GSoC2018* editions.
```

Comment:
Lets remove the unnecessary `,` after `GSoC2017`

Reply:
Actually the [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) can be [useful](https://www.theguardian.com/books/2017/mar/16/oxford-comma-helps-drivers-win-dispute-about-overtime-pay), though I guess in this case it doesn't really make a difference.